FBControlCore/Utility/FBControlCoreLogger.m

```diff
@@ -10,6 +10,9 @@
 #import "FBControlCoreLogger.h"
 #import "CalabashUtils.h"
 #import <asl.h>
+#import <CocoaLumberjack/CocoaLumberjack.h>
+
+static const DDLogLevel ddLogLevel = DDLogLevelDebug;
```

Comment:
maybe i'm missing something, but it doesn't appear that you added any uses of this ?